templates/default/unmount-fs.sh.erb

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
```

Comment:
should this file not be called 'umount' to be consistent with Linux?

Reply:
why do you comment out 'set -e'?
If /bin/fusermount returns an error, so should this script, right?